keyserver/README.md

```diff
@@ -0,0 +1,46 @@
+# CONIKS Server implementation in Golang
+
+## Usage
+```
+⇒  go install github.com/coniks-sys/coniks-go/keyserver/coniksserver
+⇒  coniksserver -h
+.---------------------------------.
+|                    ;11i;,       |
+|                 TIW@BC.         |
+|                C@@@2            |
+|            .t ,@WD              |
+|             21tWD.              |
+|              C I;;:             | CONIKS is a key management system
+|            .1,.I7I77,           | that provides transparency and privacy
+|        .:77:  .T 0o;cZi.        | for end-user public keys. CONIKS protects
+|      ,I7,  ,i.,2i UZ77TO2,      | end-to-end encrypted communications
+|     II  .7I;. ,I.C KTocoCNS.    | against malicious or compromised
+|    T7  1t,  ;:,A,.I;BCZCZCmW,   | communication providers and surveillance
+|   7I .2:  7T: ,7i:,7mpODUDOBW   | by storing users' encryption keys in
+|   S  Z, ,C;   Zm T.;SNGpGpmB@   | tamper-evident and publicly auditable key
+|   c; Z, D.   oIm;.I:SBmmmKN@2   | directories on the server side.
+|    U. U T   2.,:i Z7@NANB@@B    | This allows messaging clients to verify the
+|     o;,I:0  ;,,,,IGBW@W@W@o     | identity of users automatically, and
+|      ,;;oZp7.;C7E@@W@W@N7       | prevents malicious/compromised servers
+|           ,1o:T@@O7:            | from hijacking secure communications
+.---------------------------------. without getting caught.
```

Comment:
I think it's an ascii onion.  But, yeah, we should probably stick to CONIKS branding.